src/Stopwatch.php

```diff
@@ -81,6 +81,15 @@ public function isRunning(): bool
         return $this->startTime !== null;
     }
 
+    public function getLapTime(): Duration
+    {
+        if ($this->startTime === null) {
+            return Duration::zero();
```

Comment:
Actually I looked for other `Stopwatch` implementations, and stumbled upon the [JBoss](https://docs.jboss.org/jbossas/javadoc/4.0.4/common/index.html?org/jboss/util/StopWatch.html) one in Java, that looks like a good one. This one memorizes the lap duration after the stopwatch was stopped, and returns the previous value and not zero.

Rather that rushing to add this feature, I think I'd rather pass on this one until I have a good idea of what a good implementation would be. Would you mind removing it for now? Sorry for the waste of time :confused:

Reply:
No problem, I've removed that method and tried to address your other comments, let me know if I missed something, thanks.